google_cse.module

```diff
@@ -221,6 +221,9 @@ function google_cse_form_search_form_alter(&$form, &$form_state, $form_id) {
     google_cse_sitesearch_form($form);
     $form['#attributes']['class'][] = 'google-cse';
   }
+
+  google_cse_sitesearch_form($form);
+  $form['#attributes']['class'][] = 'google-cse';
```

Comment:
This executed above already and conditionally!!!
suppose this 2 lines should be removed at all